docs/user/index.rst

```diff
@@ -62,156 +62,113 @@ Read the Docs: documentation simplified
+  Using documentation tools like Sphinx or MkDocs for the first time or importing an existing project?
+  We have the tutorials to get you started!
 
-* **Connecting with GitHub, Bitbucket, or GitLab**:
-  :doc:`Connecting your VCS account </connected-accounts>`
-
-* **Read the Docs build process**:
-  :doc:`Configuration reference </config-file/index>` |
-  :doc:`Build process </builds>` |
-  :doc:`Build customization </build-customization>` |
-  :doc:`/environment-variables` |
-  :doc:`/reference/environment-variables` |
-  :doc:`/badges`
-
-* **Troubleshooting**:
-  :doc:`/support` |
-  :doc:`/faq`
+Explanation
+-----------
 
+Get a high-level overview of our platform:
 
-Advanced features of Read the Docs
-----------------------------------
+.. Descriptions here are focused on learning
 
-Read the Docs offers many advanced features and options.
-Learn more about these integrations and how you can get the most
-out of your documentation and Read the Docs.
+.. TODO: This next item needs its article to be finished in a separate PR
+.. https://github.com/readthedocs/readthedocs.org/pull/10071
 
-* **Advanced project configuration**:
-  :doc:`subprojects` |
-  :doc:`Single version docs <single-version>` |
-  :doc:`flyout-menu` |
-  :doc:`feature-flags`
+💡 :doc:`Continuous Documentation </integrations>`
+  Discover the advantages of having your documentation continuously deployed.
 
-* **Multi-language documentation**:
-  :doc:`Translations and localization <localization>`
+💡 :doc:`/choosing-a-site`
+  Learn about the differences between |org_brand| and |com_brand|.
 
-* **Versions**
-  :doc:`Automation rules <automation-rules>`
+.. TODO: This next item needs its article to be finished in a separate PR
+.. https://github.com/readthedocs/readthedocs.org/pull/10071
+.. TODO: 💡 Advanced topics: Deep-dive into Read the Docs
+..  Get familiar with some of the more advanced topics of building and deploying documentation with Read the Docs.
 
-* **Topic specific guides**:
-  :doc:`How-to guides <guides/index>`
 
-* **Extending Read the Docs**:
-  :doc:`REST API <api/index>`
+How-to guides
+-------------
 
+Need to get something specific done? These guides provide step-by-step instructions on various areas:
 
+.. Descriptions here are active, learn, setup, etc.
 
-Read the Docs for Business
---------------------------
+🪄 :doc:`/guides/cross-referencing-with-sphinx`
+  Learn how to use cross-references in a Sphinx project.
 
-Read the Docs has a commercial offering with improved support and additional features.
+🪄 :doc:`/analytics`
+  Learn more about how users are interacting with your documentation.
 
-* **Read the Docs for Business**:
-  :doc:`Organizations <commercial/organizations>` |
-  :doc:`Single sign on <commercial/single-sign-on>` |
-  :doc:`Project privacy level <commercial/privacy-level>` |
-  :doc:`Sharing externally <commercial/sharing>`
+🪄 :doc:`/guides/pull-requests`
+  Setup pull request builds and enjoy previews of each commit.
```

Comment:
I was going for the broad sample size to show some width in the how-to section and implicitly in our features.

Reply:
But I also found it very time-consuming to pick the right sample of how-tos :)

Reply:
The reason that the cross-reference guide was at the top was more because it's the second-most popular guide :)

![image](https://user-images.githubusercontent.com/374612/228050021-6cbbb7c1-99dc-4f5e-978c-3c64f9b230f9.png)

I'll swap it with pull requests for now, but would be nice to figure out this in the next steps :+1:

Reply:
I've swapped them now! Let's make it better in the next iteration, I think that there's a couple of factors to add, not least that we haven't considered the layout yet. This change is mostly focused on reflecting the Diátaxis structure (and being better than the previous wall of links).

Reply:
Yea... there's more to do here, but not in this PR :) 

We should eventually have the sidebar & front page order match, but let's move forward for now.